scripts/variantstore/wdl/GvsExtractCallset.wdl

```diff
@@ -140,6 +140,9 @@ task ExtractTask {
 
     String mode
 
+    Boolean emit_pls
+    Boolean emit_ads
+
```

Comment:
sorry that I wasn't clear —  I don't think these need to be parameterized, I think the "true" and "false" values can just be passed directly in the call to ExtractTask (lines 96 + 97).